packages/orbit-components/src/Accordion/AccordionSection/components/SectionContent.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import defaultTheme from "../../../defaultTheme";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: ${({ theme }) => `0 ${theme.orbit.spaceLarge} ${theme.orbit.spaceLarge}`};
```

Comment:
You can use `Stack` for same functionality

Reply:
What's the status of this?

Reply:
@silvenon @mainframev  I've removed `display-flex` and `flex-direction` but I'm not sure how to add padding on Stack component 🤔

Reply:
you had padding on right side and bottom, Stack will add only on bottom automatically, when they align as column. Probably you don't need that right padding inside SectionContent, consider to add it inside `Accordion` 🤔

Reply:
I can use `Stack` for bottom spacing 👍, but I need to keep left-right padding inside this component because of sticky footer component which needs 100% width.